packages/mc-scripts/src/utils/credentials-storage.js

```diff
@@ -1,24 +1,35 @@
-const { existsSync, writeFileSync, readFileSync, mkdirSync } = require('fs');
+const fs = require('fs');
+const path = require('path');
 const homedir = require('os').homedir();
 
-const dirPath = `${homedir}/.commercetools`;
-const filename = 'mc-credentials.json';
-const filePath = `${dirPath}/${filename}`;
+const credentialsFolderPath = path.join(homedir, `.commercetools`);
+const credentialsFilePath = path.join(
+  credentialsFolderPath,
+  'mc-credentials.json'
+);
 
 class CredentialsStorage {
+  static location = credentialsFilePath;
+
+  constructor() {
+    // Ensure the credentials file is present
+    if (!fs.existsSync(credentialsFilePath)) {
+      fs.mkdirSync(credentialsFolderPath, { recursive: true });
+      // Initialize with an empty object
+      this._writeCredentials({});
+    }
+  }
```

Comment:
There was a small issue when the folder `.commercetools` was present but not the file.

So I decided to move the logic of initializing the file to the constructor.